db/migrate/20171026150119_rename_disabled_to_anyone_disabled.rb

```diff
@@ -0,0 +1,19 @@
+class RenameDisabledToAnyoneDisabled < ActiveRecord::Migration[5.1]
+  def up
+    add_column :medicaid_applications, :anyone_disabled, :boolean
```

Comment:
should we default to `false` ? can't remember if we do that for other similar fields

Reply:
I don't think we should and here's why - if we have `nil` for these answers we know they have not been answered. Whereas a false or true will tell us they've been answers. I think it's important to be able to infer if a client has gone through/by that step and if we default to a value, we'd lose that visibility.